assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -2437,7 +2674,28 @@ tests:
               relation: viewer
             expectation:
               - document:public
-
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:public
+              relation: viewer
+            expectation:
+              - user:*
```

Comment:
This should be `[user:*, user:bob]`.

user:bob is part of the FGA group which is assigned to document:public#viewer in addition to the wildcard being assigned to `document:public#viewer` as well.